### DISCUSSION TITLE
Unable to resolve "graphql-request" from "src\app\index.jsx"

### DISCUSSION THREAD

**Discussion by abegy002** [nrwl/nx#26312] [General]
I encountered an error message stating "Unable to resolve 'graphql-request' from 'src\app\index.jsx'" while working on my project. This issue seems to be related to the inability of the module resolver to locate the 'graphql-request' package. I've checked my project's dependencies, and 'graphql-request' is indeed installed. However, it appears that the module resolution configuration may be incorrect. Could someone provide guidance on how to troubleshoot and resolve this issue?

**Comment by saheedsh2** [comment]
You can solve this by going to your package.json file:
change your graphql to : 15.8.0
and graphql-request: 6.1.0
it works trust me

**Reply by 0686153873** [reply]
the problem is still . is there any solution for that ?

**Reply by saheedsh2** [reply]
Did u do npm install after editing the package.json file?
